psalm-baseline.xml

```diff
@@ -1,3 +1,24 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<files psalm-version="5.0.0@4e177bf0c9f03c17d2fbfd83b7cc9c47605274d8">
+<files psalm-version="5.26.1@d747f6500b38ac4f7dfc5edbcae6e4b637d7add0">
+  <file src="src/Input/AbstractParamAwareInput.php">
```

Comment:
Is it easy enough to fix the risky comparisons instead of baselining them?

Reply:
done